src/accordian/Accordion.tsx

```diff
@@ -13,11 +13,19 @@ import {classNames} from "../utils";
 import {ClassNames} from "./ClassNames";
 import {Icon, Direction} from "../icon";
 
+/**
+ * Accordian Props
+ * style: css style
+ * className: css property
+ * accordianMultiPanel: if yes multipanel enebled else not
+ * dataqa: quality engineering testing field
```

Comment:
use below format for documentation
@param {dataqa} quality engineering tag

Reply:
Addressed